src/osd/PrimaryLogPG.cc

```diff
@@ -1848,7 +1848,8 @@ void PrimaryLogPG::do_op(OpRequestRef& op)
 	     << *m << dendl;
     return;
   }
-  if (!(m->get_source().is_mds()) && osd->check_failsafe_full() && write_ordered) {
+  if (!(m->get_source().is_mds()) && osd->check_failsafe_full() && 
+      !m->has_flag(CEPH_OSD_FLAG_FULL_FORCE) && write_ordered) {
```

Comment:
The ops of deleting an image will be discarded here if don’t touch the failsafe condition